app/schemas/archive-format.json

```diff
@@ -0,0 +1,222 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://the-federation.info/specs/backup-restore#archive_format",
```

Comment:
So, what's the status of this document? Is it even maintained? I see a lot of opened issues in the gitlab and nobody seems to care.